worldedit-bukkit/adapters/adapter-1_18_2/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_18_R2/PaperweightFaweAdapter.java

```diff
@@ -547,6 +562,163 @@ protected ServerLevel getServerLevel(final World world) {
         return ((CraftWorld) world).getHandle();
     }
 
+    @Override
+    public boolean generateFeature(ConfiguredFeatureType feature, World world, EditSession editSession, BlockVector3 pt) {
```

Comment:
Could this be moved into FaweAdapter similar to generateTree?

Reply:
See https://github.com/IntellectualSites/FastAsyncWorldEdit/pull/2364#discussion_r1403098520

Reply:
*Oh*.
I guess it's fine then, maybe I'll try to extract parts of it when working on Folia stuff, but it shouldn't be a blocker here.

Reply:
It would just leave a method in FaweAdapter with about 5 lines in, and then the block placement separate really